torchao/csrc/cuda/sparse_marlin/base.h

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Make sure to also create a test here https://github.com/pytorch/ao/tree/main/test for the cuda kernel and a test for an end to end flow https://github.com/pytorch/ao/tree/main/test/sparsity